### PR TITLE
Add Spec

### PR DESCRIPTION
**Pull request by droogmic**
Add JSON Spec
Valdate JSON spec in tests
Accidentally reformatted the tests

### CODE REVIEW

**Comment by droogmic** [comment]
I have read https://github.com/a-b-street/osm2lanes/issues/3, and I don't really know what to do, but encoding what already exists doesn't seem like a terrible idea...

**Comment by dabreegster** [comment]
Looks good to me -- we never really landed on any decision about how to express the schema, or about how it should handle all the nuance mentioned in the issue. But I agree that encoding what we're doing right now is a solid step forwards. Looks good to go, @enzet?

File: .github/workflows/test.yml [hunk 1]
```diff
@@ -21,7 +21,9 @@ jobs:
     - name: Install dependencies
       run: |
         python -m pip install --upgrade pip
-        pip install pytest
+        python -m pip install --requirement requirements.txt
```
Comment (enzet) [inline .github/workflows/test.yml:24]: Great! I should have added `requirements.txt` in the first place.

**Comment by enzet** [comment]
Sorry, my bad. It turned out reviews aren't visible until I submit them. :facepalm:

**Pull request merged**